src/templates/_components/fieldtypes/Assets/input.html

```diff
@@ -20,7 +20,7 @@
 
     {% if sources %}
         <div class="flex{% if limit and elements|length >= limit %} hidden{% endif %}">
-            <button type="button" class="btn add icon dashed">{{ selectionLabel }}</button>
+            <button type="button" class="btn add icon dashed" aria-label="{{ selectionLabel }}">{{ selectionLabel }}</button>
```

Comment:
This should probably be made to `_includes/forms/elementSelect.html` so it affects Entries and Users fields as well.

Reply:
@brandonkelly have made that change!